smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestRequestGenerator.kt

```diff
@@ -75,20 +76,26 @@ open class HttpProtocolUnitTestRequestGenerator protected constructor(builder: B
     }
 
     private fun renderClientBlock(test: HttpRequestTestCase) {
```

Comment:
Client creation is cleaned up below by supplying all config fields as initializer params instead of setting them after config object creation.